src/append.js

```diff
@@ -12,9 +12,9 @@ var _curry2 = require('./internal/_curry2');
  * @category List
  * @sig a -> [a] -> [a]
  * @param {*} el The element to add to the end of the new list.
- * @param {Array} list The list whose contents will be added to the beginning of the output
+ * @param {Array} list The list of elements to appenda new item to.
```

Comment:
There's a typo: "appenda". I'd prefer there was a way to express this without reusing "append" in the description, but not a huge deal http://www.thesaurus.com/browse/append?s=t